### DISCUSSION TITLE
Using onedrive to backup

### DISCUSSION THREAD

**Discussion by thewalkerz** [abraunegg/onedrive#2038] [Q&A] [answered]
Hi, I have a little questions, is possible to use onedrive to backup a specific folder on my linux machine? I would like to use to backup my Documents, images and videos from my linux machine. I read the config guide but only show me the sync_dir only one way.

**Comment by abraunegg** [comment, marked as answer]
@thewalkerz 
> Hi, I have a little questions, is possible to use onedrive to backup a specific folder on my linux machine? I would like to use to backup my Documents, images and videos from my linux machine. I read the config guide but only show me the sync_dir only one way.

The configuration item 'sync_dir' is where you have your `~/OneDrive` folder located.

You could configure this option to be anywhere else on your system - for example: `sync_dir = "/some/other/folder/"`

To backup your data then to OneDrive you have the following options:
* Archive all your 'folders' you want to backup to a tar.gz file and move this archive to your 'sync_dir'
* Symbolically link all the folders you want to sync to OneDrive inside your 'sync_dir' so that the data is replicated to OneDrive

Always test with `--dry-run` to ensure the client will do what you are expecting before performing an actual sync.
